### DISCUSSION TITLE
Can I access Qfield Cloud data at local directory in ios ?

### DISCUSSION THREAD

**Discussion by JUJUBAEDATE** [opengisch/QField#2913] [Q&A]
My Qfield project is using orthomosaic map and has many attached photos.
So I wonder if there is any way to access QfieldCloud data at local in IOS.

In my Android 12 smartphone, I can access imported project directly(using usb cable) At '/Android/data/ch.opengis.qfield/imported project/'.

I know Ios app has each sandbox data container. When connecting Ipad with PC by imazing, it said 'file sharing' enabled is needed.

**Comment by JUJUBAEDATE** [comment]
I just found solution about only "Download" project data to my pc by using 'Back Up'. But still I can't put files into my ipad Qfield app library>.....>cloud_projects folder.

**Reply by ryanpeel** [reply]
Hello.  I am in need of getting files off of an ipad that can no longer synchronize.  How did you end up doing this on ios?

**Reply by bloqbreaker** [reply]
I just encountered the problem of not being able to synchronize as well, in my case I got an error message of 'unknown error', and found a makeshift solution:
1. install iTunes
2. in iTunes: select the device and back it up - the data created by this is machine readable only, some kind of encryption apple uses
3. install iMazing - a program that allows to browse iOS backup files and shows the backups folder structure
4. open iMazing - in my case it found the backup file automatically. Locally stored backups even are sorted by device.
5. select the device, which's backup you want to access and navigate to 'Data' on the left side tab
6. select Apps
7. search for QField in the list off apps and navigate the folder structure to 'Library/Application Support/OPENGIS.ch/QField/cloud_projects/ [QField account name]'
8. from here you'll have to search through the folders for your project
9. select folder and click on 'copy to PC', select the directory on your PC you want to save it to
10. celebrate! you got your data back :)

With the iMazing trial version you got 10 folders you can copy to your PC. In my case two were enough to find the data, though for future projects I wish to see the QField Cloud sync problem with iOS fixed.

Be welcome to share this solution with others! Just please always refer to the original post ;)
